docs/src/docs/asciidoc/user/107.adoc

```diff
@@ -9,10 +9,10 @@ ifdef::notBuildingForSite[]
 include::menu.adoc[]
 endif::notBuildingForSite[]
 
-== Getting started with Ehcache 3.0 & JSR-107
+== Getting started with Ehcache 3 & JSR-107
 
-There is nothing special for you to do to use Ehcache 3.0 as the caching provider for your application: add the
-`ehcache-3.0.0.jar` to your application's classpath (possibly removing the previous provider's jar) and you are ready to
+There is nothing special for you to do to use Ehcache 3 as the caching provider for your application: add the
+`ehcache-3.1.0.jar` to your application's classpath (possibly removing the previous provider's jar) and you are ready to
 go:
```

Comment:
should we think about putting `ehcache-3.1.0.jar` on it's own line - futureproofing?